sessionize/app/src/main/java/co/touchlab/sessionize/sponsors/SponsorSessionAdapter.kt

```diff
@@ -72,8 +72,8 @@ class SponsorSessionAdapter(private val activity: Activity) : RecyclerView.Adapt
     override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) {
         when (EntryType.values()[holder.itemViewType]) {
             EntryType.TYPE_HEADER -> {
```

Comment:
We should probably remove the header type and functionality rather than just commenting out here